pages/contact.html

```diff
@@ -65,24 +66,72 @@
-            <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a>
-        </p>
-        <p>Web: mountsioncbssecondary.ie</p>
-        <p>Telephone Number: 051 377378</p>
-        <p>Fax Number: 051-376468</p>
-        <p>School Roll No. 64930i</p>
-        <p>Address:</p>
-        <p>Mount Sion CBS Secondary School,</p>
-        <p>Barrack Street,</p>
-        <p>Waterford City.</p>
-        <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
-        <p>School Location on Google maps</p>
-        <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739"
-            width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
-    </div>
-</body>
+<div class="ui main text container">
+    <h1 class="ui dividing header" style="padding-top:8px">School Contact Details</h1>
+    <p>Email School: <a href="mailto:info@mountsioncbssecondary.ie" target="_top">info@mountsioncbssecondary.ie</a></p>
+    <p>Web: mountsioncbssecondary.ie</p>
+    <p>Telephone Number: 051 377378</p>
+    <p>Fax Number: 051-376468</p>
+    <p>School Roll No. 64930i</p>
+    <p>Address:</p>
+    <p>Mount Sion CBS Secondary School,</p>
+    <p>Barrack Street,</p>
+    <p>Waterford City.</p>
+    <h1 class="ui dividing header" style="padding-top:8px">School Location</h1>
+    <p>School Location on Google maps</p>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d9769.151915664062!2d-7.1166274!3d52.2563154!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0%3A0xf8355ef4efe9f1a4!2sMount+Sion+CBS+Secondary+School!5e0!3m2!1sen!2sie!4v1523822538739" width="100%" height="450" frameborder="0" style="border:0" allowfullscreen></iframe>
+</div>
+<br/>
+
+<div class="ui inverted blue vertical footer segment">
+    <div class="ui center aligned container">
+      <div class="ui inverted section divider"></div>
+        <div class="ui stackable divided grid">
+            <div class="four wide column">
+                <div class="ui link list">
+                    <a href="https://mountsion.vsware.ie/Login.jsp" class="item">
+                        Vs Ware
+                    </a>
+                    <a href="mailto:info@mountsioncbssecondary.ie" target="_top" class="item">
+                        Email
+                    </a>
+                    <a href="https://www.education.ie/en/" class="item">
+                        Department of Education
+                    </a>
+                    <a href="https://www.examinations.ie" class="item">
+                        State Exams Commission
+                    </a>
+                </div>
+            </div>
+            <div class="four wide column">
+                <div class="ui link list">
+                    <a href="http://www.pdst.ie" class="item">
```

Comment:
Also as pointed out these are hard to read, this is because they are hyperlinks and are usually a bluish color by standard, if you go and try to override the `<a>` tag it will change for every hyperlink on the site, I would recommend maybe using some inline styling or else make a css class for them. I'll leave it up to you,

Reply:
Awesome that works, good catch